Userland/Libraries/LibWeb/HTML/BrowsingContextContainer.cpp

```diff
@@ -48,6 +48,9 @@ const DOM::Document* BrowsingContextContainer::content_document() const
 
     // 3. Let document be context's active document.
     auto const* document = context.active_document();
+    if (document == nullptr) {
+        return nullptr;
+    }
```

Comment:
Hmm, I don't think this is right - https://github.com/SerenityOS/serenity/blob/9521f719440d3ecbb5f06dbd86ca646f14bf862b/Userland/Libraries/LibWeb/DOM/Node.cpp#L711 shouldn't be using `content_document()` in the first place, but `content_document_without_origin_check()` below. Please revert this.

Additionally, please don't add non-standard steps like this without a comment, given that everything else has one :)

Reply:
Just to follow up on this.  Right now in that function we call VERIFY on the document and crash the browser if it fails.  This seems bad to me, though is it there for a reason?  I could imagine that the VERIFY there shows us where there may be other bugs in LibWeb that are causing an empty document?  If so that does seem useful, but I just wanted to check in about that, since catching the missing document object here and returning nullptr seems cleaner to me.  Though I'm not sure why the document object is null in the first place with empty <iframe> elements.